internal/services/managedapplications/client/client.go

```diff
@@ -1,24 +1,31 @@
 package client
 
 import (
-	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2019-07-01/managedapplications" // nolint: staticcheck
```

Comment:
looks like there's still a reference to this/we'll need to do a `go mod tidy` / `go mod vendor` here?